client/dashboard/sites/deployments/index.tsx

```diff
@@ -1,16 +1,72 @@
+import { useQuery } from '@tanstack/react-query';
+import { __experimentalText as Text } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { siteQuery } from '../../app/queries';
+import { siteRoute } from '../../app/router';
+import { Callout } from '../../components/callout';
+import { CalloutOverlay } from '../../components/callout-overlay';
 import DataViewsCard from '../../components/dataviews-card';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import RouterLinkButton from '../../components/router-link-button';
+import { SitePlan } from '../../data/types';
+import illustrationUrl from './deployments-callout-illustration.svg';
+import ghIconUrl from './gh-icon.svg';
 
 function SiteDeployments() {
+	const { siteSlug } = siteRoute.useParams();
+	const { data: site } = useQuery( siteQuery( siteSlug ) );
+
+	if ( ! site ) {
+		return;
+	}
+
+	const showCallout =
+		( ! site.plan || ! hasDeploymentsFeature( site.plan ) ) &&
+		// Show to all atomic sites regardless of plan status e.g. an atomic site in the revert queue
+		// after a plan downgrade will continue to see Deployments until the revert is complete
+		! site.is_wpcom_atomic;
```

Comment:
The condition seems to be different from the original logic in v1:

https://github.com/Automattic/wp-calypso/blob/9eca4fcfea8d0fadd390c7573f9540a12e23d6e7/client/sites/components/site-preview-pane/dotcom-preview-pane.tsx#L67

`showCallout` should be the inverse of the above condition. 🤔

Reply:
Interesting, I think it's sort of annoying when an atomic site can't see the atomic features. But that might just be a dev thing 😃 It makes sense to copy v1 business logic unless making an explicit proposal to change it.

Reply:
IIRC the "features" in the wpcom backend code is tied to plans, not the atomic state.

We can iterate on it later when we actually implement the Deployments screen.